fastify.js

```diff
@@ -565,7 +573,11 @@ function build (options) {
 
   function addHook (name, fn) {
     if (name === 'onClose') {
+      this._hooks.validate(name, fn)
```

Comment:
The `onClose` function from `avvio` already includes validation, so this isn't necessary.

Reply:
I know but it's developer friendly when all places throw the same error messages.

Reply:
Okay that's true 👍